imap_processing/glows/l0/decom_glows.py

```diff
@@ -64,11 +65,28 @@ def decom_packets(
                 apid = packet.header["PKT_APID"].derived_value
                 # Do something with the packet data
                 if apid == GlowsParams.HIST_APID.value:
-                    hist_l0 = HistogramL0(packet, version, filename)
+                    values = [
+                        item.derived_value
+                        if item.derived_value is not None
+                        else item.raw_value
+                        for item in packet.data.values()
+                    ]
+                    hist_l0 = HistogramL0(
+                        version, filename, CcsdsData(packet.header), *values
```

Comment:
Yes, it does depend on that. Since the XTCE definition is sourced from the actual packet structure, I assumed that wouldn't be changing much. If it does change, I (or someone) would need to rearrange these attributes or explicitly define all of the parameters. However, that should be caught by the tests which check the attributes.

Reply:
> However, that should be caught by the tests which check the attributes.

That is a good point that you have tests which assert on specific values, so if those get bumped one further along then that should raise. Sounds good to me.